SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/DynamicActorAwakeChangePropertyTest.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "DynamicActorAwakeChangePropertyTest.h"
+
+#include "DormancyTestActor.h"
+
+// This test checks that changing the value of a replicated property on an actor whose NetDormancy is DORM_Awake will result in the property
+// being replicated.
+
+ADynamicActorAwakeChangePropertyTest::ADynamicActorAwakeChangePropertyTest()
+{
+	Author = "Matthew Sandford";
+}
+
+void ADynamicActorAwakeChangePropertyTest::PrepareTest()
+{
+	Super::PrepareTest();
+
+	// Step 1 - Spawn dormancy actor and check NetDormancy is DORM_Initial
+	AddStep(TEXT("ServerSpawnDormancyActor"), FWorkerDefinition::Server(1), nullptr, [this]() {
+		AActor* Actor = CreateDormancyTestActor();
```

Comment:
nit - perhaps CreateDormancyTestActor should take the dormancy type you want the actor to be initially